lib/travis/build/script/elm.rb

```diff
@@ -0,0 +1,113 @@
+module Travis
+  module Build
+    class Script
+      class Elm < Script
```

Comment:
Does it make sense to inherit from `NodeJs` class? This would allow some additional nifty features that may prove useful.